### PR TITLE
adds the comment button and starts the funds system

### PR DESCRIPTION
**Pull request by blueweaver**
This is the updated version of the website that I showcased on Wednesday

### CODE REVIEW

File: Gruvest-Django/mysite/myapp/migrations/0011_auto_20201012_0547.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# Generated by Django 3.1.1 on 2020-10-12 05:47
```
Comment (jhowell1) [inline Gruvest-Django/mysite/myapp/migrations/0011_auto_20201012_0547.py:1]: Can you explain what this segment of code is doing?

File: Gruvest-Django/mysite/myapp/views.py [hunk 2]
```diff
@@ -33,13 +34,72 @@ def form_valid(self, form):
...
+        return '/'
+
 '''
 PitchDetail inherits from DetailView
     is a Retrieve operation
 '''
 class PitchDetail(DetailView):
     template_name = "view_pitch.html"
     model = models.PostModel
+'''
```
Comment (jhowell1) [inline Gruvest-Django/mysite/myapp/views.py:91]: I know this is commented but can you explain what this is for?

Comment (blueweaver) [reply Gruvest-Django/mysite/myapp/views.py:91]: The purpose of this function is the check if the user has enough funds to purchase the ability to view the full post but I have gotten it to fully work yet.

File: Gruvest-Django/mysite/myapp/migrations/0011_auto_20201012_0547.py (hunk 1 above)
Comment (blueweaver) [reply Gruvest-Django/mysite/myapp/migrations/0011_auto_20201012_0547.py:1]: This is the newest migration file. This is how Django is able to convert the models in are models.py file into SQL code.

**Pull request merged**